demo/src/issues/4/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Issue #4 Repro</title>
+    <link rel="stylesheet" href="https://early.webawesome.com/webawesome@3.0.0-beta.6/dist/styles/themes/default.css" />
```

Comment:
I almost forgot to mention -- since I filed this issue, webawesome has released their 3.0.0 version. You might want to use `https://cdn.jsdelivr.net/npm/@awesome.me/webawesome@3.0.0/dist/styles/themes/default.css` (and similar for the other files) instead of the beta version.

Reply:
i tried, but i get Uncaught TypeError: Failed to resolve module specifier "@shoelace-style/animations". Relative references must start with either "/", "./", or "../".` when trying to load. anywhy it is only repro so its fine (the root cause already identified and added as additional example to the demo)